.cursorignore

```diff
@@ -0,0 +1 @@
+# Add directories or file patterns to ignore during indexing (e.g. foo/ or *.csv)
```

Comment:
Do we need to add some directories here? Like the raw data or installed packages in the `venv` folder?